main.go

```diff
@@ -97,9 +102,11 @@ func (s ParcelService) Delete(number int) error {
 }
 
 func main() {
-	// настройте подключение к БД
-
-	store := // создайте объект ParcelStore функцией NewParcelStore
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		fmt.Printf("error open database: %v", err)
```

Comment:
Нужно либо добавить выход из функции или сделать так
```suggestion
		lof.Fatalf("error open database: %v", err)
```